tools/utils.py

```diff
@@ -111,7 +111,13 @@ def IsWindows():
 
 
 def SearchFiles(dir, ext):
-  list = glob.glob(dir+ '/**/*.' + ext, recursive=True)
+  list = []
```

Comment:
`list` is a Python builtin so it should not be shadowed.  Instead, could we use `valid_files` or similar?